src/api/common/schemas/quality.json

```diff
@@ -1,8 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "quality",
-  "description": "Ratio for incoming/outgoing transit fees represented in billionths. (For example, a value of 500 million represents a 0.5:1 ratio.) As a special case, 0 is treated as a 1:1 ratio.",
-  "type": "integer",
-  "minimum": 0,
-  "maximum": 1000000000
+  "description": "Ratio for incoming/outgoing transit fees. As a special case, 0 is treated as a 1:1 ratio.",
+  "type": "number",
+  "minimum": 0
```

Comment:
I asked in the development channel if there was a similar limit for quality, but I haven't gotten an answer yet. I may just assume that it has the same limit. I don't think our schema validator properly supports checking for the correct number of decimal places, though I believe json schema supports it with the `multipleOf` property. Maybe I'll submit a bug report.

The only reason we are using strings for currency values is because javascript floats don't have enough precision for them. So for numbers where they do have enough precision, there's no need to use strings. One could argue for consistency or one could argue for minimizing the "damage" and keeping it limited to just currency values. Currently it is choosing the latter.